box-ui/src/components/css/BoxMeeting.css

```diff
@@ -29,3 +29,61 @@
 .JitsiComponent {
     flex-grow: 3;
 }
+
+.containerStudent {
```

Comment:
Those rules should actually be in a separate `StudentMeeting.css` and we could perhaps create an global `Meeting.css`stylesheet for rules used in all the interfaces